src/ds323x/alarms.rs

```diff
@@ -170,6 +189,18 @@ where
         self.iface.write_data(&mut data)
     }
 
+    /// Set Alarm2 for a time every day.
```

Comment:
Same as above, could you add a description of the matching strategy (i.e. fires when h, m match)